src/components/partnership/partnership.service.ts

```diff
@@ -621,6 +623,28 @@ export class PartnershipService {
           filter,
         },
       });
+
+      await Promise.all(
+        result.items.map(async (item) => {
+          const query = `
+              MATCH (partnership:Partnership {id: $id, active: true})
+                -[:organization {active: true}]->(organization)
+              RETURN organization.id as id
+            `;
+          const orgId = await this.db
+            .query()
+            .raw(query, {
+              id: item.id,
+            })
+            .first();
+
+          const org = await this.orgService.readOne(orgId?.id, session);
+          Object.assign(item, {
+            ...item,
+            organization: org,
+          });
+        })
+      );
```

Comment:
Mutating like this is not a good pattern. It's better to create a new object and new array with those changes.
```suggestion
      const items = await Promise.all(
        result.items.map(async (item) => {
          const query = `
              MATCH (partnership:Partnership {id: $id, active: true})
                -[:organization {active: true}]->(organization)
              RETURN organization.id as id
            `;
          const orgId = await this.db
            .query()
            .raw(query, {
              id: item.id,
            })
            .first();

          const org = await this.orgService.readOne(orgId?.id, session);
          return {
            ...item,
            organization: org,
          };
        })
      );
```